corehq/apps/userreports/management/commands/build_report_builder_v2_diffs.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function
+import logging
+import os
+import filecmp
+import difflib
+
+import re
+from django.core.management import BaseCommand
+
+
+logger = logging.getLogger('app_migration')
+logger.setLevel('DEBUG')
+
+VERSIONED_DIRS = [
+    # List of report builder v1 directories.
+    # Paths are relative to corehq/apps/userreports
+    "v1/",
+    "reports/builder/v1/",
+    "static/userreports/js/v1/",
+    "templates/userreports/v1/",
+    "templates/userreports/partials/v1/",
+    "templates/userreports/reportbuilder/v1/",
+]
+
+RELATIVE_DIFF_STORAGE = 'tests/data/report_builder_v2_diffs/'
+
+
+def get_diff_filename(v2_dir, filename):
+    dir_part = ".".join([x for x in v2_dir.split("/") if x and x != ".."])
```

Comment:
nit: could use generator expression instead of list comprehension here (no need to create intermediate list)